core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/components/dot-edit-content-sidebar-locales/dot-edit-content-sidebar-locales.component.ts

```diff
@@ -53,6 +63,38 @@ export class DotEditContentSidebarLocalesComponent {
      */
     switchLocale = output<DotLanguage>();
 
+    readonly #dotMessageService = inject(DotMessageService);
+
+    $maxLocaleChips = signal(9);
```

Comment:
@hmoreras I think 9 should be a constanst like a `MAX_LOCALES = 9`

Reply:
done